engines/bastion_katello/test/content-hosts/bulk/content-hosts-bulk-action-subscriptions.controller.test.js

```diff
@@ -28,9 +28,10 @@ describe('Controller: ContentHostsBulkActionSubscriptionsController', function()
         CurrentOrganization = 'foo';
     });
 
-    beforeEach(inject(function($controller, $rootScope, $q) {
+    beforeEach(inject(function($injector, $controller, $rootScope, $q) {
         $_q_ = $q;
         $scope = $rootScope.$new();
+        GlobalNotification = $injector.get('GlobalNotification');
```

Comment:
Problem is that we have two different styles and some tests use the `$injector` style while others use the named dependency style.